README.md

```diff
@@ -172,7 +172,23 @@ We're going to try spinning this out into its own library; see: https://github.c
 			<script src="../node_modules/heatmap.js/build/heatmap.min.js"></script>
 			<script src="../node_modules/leaflet-heatmap/leaflet-heatmap.js"></script>
 
+## To use overlapping marker spiderfier plugin:
+  
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/OverlappingMarkerSpiderfier-Leaflet/0.2.6/oms.min.js"></script>
+      var oms = omsUtil(map, {
+            keepSpiderfied: true,
+            circleSpiralSwitchover: 0
+      });
```

Comment:
@crisner ...What do you think of this? Are we missing anything? Thanks!

Reply:
Can we hold this section? The function omsUtil is a custom function(not part of the plugin) and there is no reference to it here. I will need to take a look at it.

Reply:
Sorry for the confusion. Looks good @sagarpreet-chadha. I don't think we are missing anything :)

Reply:
Perhaps we could give more control to the user by listing the options they could use? For example,
```
var oms = omsUtil(map, options);
```
And then document the options from https://github.com/jawj/OverlappingMarkerSpiderfier-Leaflet
What do you think?